src/Extractors/LatteExtractor.php

```diff
@@ -46,7 +46,6 @@ public function extractFile($file, MessageCatalogue $catalogue)
 	{
 		$buffer = NULL;
 		$parser = new Parser();
-		$parser->shortNoEscape = TRUE;
```

Comment:
Is it ok to remove this for Latte 2.4?

Reply:
For 2.4 absolutely, it was already deprecated then. For lower versions I'm not sure, but nobody objected when I asked for review here and on Slack.

Reply:
composer.json here requieres `"latte/latte": "^2.4.6@dev",` anyway so we don't need to handle the lower versions